README.md

```diff
@@ -260,138 +260,168 @@ We gladly credit all contributors.
 
 ### What is equity worth?
 
-- [**Public companies**](https://en.wikipedia.org/wiki/Public_company) are corporations where any
-  member of the public can own stock.
+🚧 This section needs to establish a relationship between each of these definitions and
+the concept of what equity is worth.
+How does each of these circumstances affect the “worth” of shares?
+May also consider whether this section should be “What is the value of equity?”, to
+parallel the section that follows.
+
+- 🄳 [**Public companies**](https://en.wikipedia.org/wiki/Public_company) are corporations where
+  any member of the public can own stock.
   People can buy and sell the stock for cash on public
   [exchanges](https://www.investopedia.com/terms/e/exchange.asp). The value of a company’s
   shares is the value you see in the stock market reports, so shareholders know how much
   their stock is worth.
-- Most smaller companies, including all startups, are
+- 🄳 Most smaller companies, including all startups, are
   [**private companies**](https://en.wikipedia.org/wiki/Privately_held_company) with owners who
-  control its operation.
+  control how those companies operate.
   Unlike a public company, where anyone is able to buy and sell stock, owners of a private
   company control who is able to buy and sell their stock.
   There may be few or no transactions, or they may not be publicly known.
 - ❗️ It’s hard to know the “real value” of private company stock.
   Because the value of private company stock is not determined by public trades,
   shareholders can only make educated guesses about the likely future value, at a time when
   they will be able to buy or sell the stock.
-  It’s always worth remembering stock is a legal agreement that entitles you to something of
-  highly uncertain value, and could well be worthless in the future, or highly valuable,
+  It’s always worth remembering *stock is a legal agreement* that entitles you to something
```

Comment:
The point of this sentence was the highly uncertain value (not the legal agreement), so not sure this is what we should emphasize. I'll rewrite.